src/sentry/api/serializers/models/rule.py

```diff
@@ -43,5 +54,6 @@ def serialize(self, obj, attrs, user):
             obj.label,
             'dateCreated':
             obj.date_added,
+            'environment': environment.name if environment is not None else None,
```

Comment:
What kind of monster auto-formatter formatted the lines above this one v__v

Reply:
lol its so bad